hoot-core/src/main/cpp/hoot/core/io/OsmApiChangeset.cpp

```diff
@@ -713,7 +713,7 @@ bool XmlChangeset::isSent(XmlElement* element)
   else
     //  Sent means Buffering, Sent, or Finalized
     return element->getStatus() == XmlElement::ElementStatus::Buffering ||
-           element->getStatus() == XmlElement::ElementStatus::Sent ||
+           (element->getStatus() == XmlElement::ElementStatus::Sent && element->id() > 0) ||
```

Comment:
An element with a negative ID that is sent cannot be marked as sent yet because it needs a positive ID so that it can be used elsewhere.  If it is just buffered (in this changeset already) then it is fine.